corpus/expressions.txt

```diff
@@ -378,7 +378,7 @@ yield from a
       (expression_list (identifier))
       (yield (expression_list (integer)))))
   (expression_statement
-    (yield (identifier))))
+    (yield (expression_statement (identifier)))))
```

Comment:
I think it's weird to call these things 'expression statements'. Conceptually, they're expressions, not statements.